logging/bin/gfuser.sh

```diff
@@ -0,0 +1,59 @@
+#! /bin/bash
```

Comment:
This script is only a testing convienence, it will NOT become permanent part of the project.  The logic included will eventually be incorporated into the create_elasticsearch_datasource*.sh script(s).